swarm.go

```diff
@@ -443,11 +453,10 @@ func (s *Swarm) Stop() error {
 	if s.ps != nil {
 		s.ps.Stop()
 	}
-	if ch := s.config.Swap.Chequebook(); ch != nil {
-		ch.Stop()
-		ch.Save()
-	}
 	if s.swap != nil {
+		//if svc := s.swap.Service; svc != nil {
```

Comment:
Either enable this code or remove it - why is it commented out?

Reply:
@vojtechsimetka do you have any input on why this was commented out? Thanks.

Reply:
I am removing it, but checking with @holisticode one detail, I will push it in a moment after checking with him

Reply:
good to be removed

Reply:
Fixed, this code was commented since Service is not a member of the swap struct anymore.